packages/url-launcher/README.md

```diff
@@ -1,7 +1,48 @@
 # url_launcher
 
-A Flutter plugin for launching a URL.
+A Flutter plugin for launching a URL in the mobile platform. Supports iOS and Android.
 
-## Getting Started
+## Usage
 
-Try out the plugin by running the project in the example/ folder.
+Get this plugin as described [here] (https://www.dartlang.org/tools/pub/get-started).
+
+### Example
+
+``` dart
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+void main() {
+  runApp(new Scaffold(
+    body: new Center(
+      child: new RaisedButton(
+        onPressed: launchURL,
+        child: new Text('Show Flutter homepage'),
+      ),
+    ),
+  ));
+}
+
+launchURL() {
+  UrlLauncher.launch('https://flutter.io');
+}
+
+```
+
+## Supported URL schemes
+
```

Comment:
Maybe start with: 'The `launch` method takes a string argument containing a URL. This URL can formatted using a number of different URL schemes.'